etc/inc/disable-common.inc

```diff
@@ -159,23 +159,23 @@ blacklist ${RUNUSER}/gsconnect
 # systemd
 blacklist ${HOME}/.config/systemd
 blacklist ${HOME}/.local/share/systemd
-blacklist /var/lib/systemd
+blacklist ${PATH}/systemctl
 blacklist ${PATH}/systemd-run
 blacklist ${RUNUSER}/systemd
-blacklist ${PATH}/systemctl
-blacklist /etc/systemd/system
 blacklist /etc/systemd/network
+blacklist /etc/systemd/system
+blacklist /var/lib/systemd
 # creates problems on Arch where /etc/resolv.conf is a symlink to /var/run/systemd/resolve/resolv.conf
 #blacklist /var/run/systemd
 
 # openrc
-blacklist /etc/runlevels/
 blacklist /etc/init.d/
 blacklist /etc/rc.conf
+blacklist /etc/runlevels/
```

Comment:
I feel like this is a separate question... but do we need the training `/`? It seems to have been inconsistently applied in this file

Reply:
Good point. Removed trailing slashes, thanks @Fred-Barclay.